purefb/metadata.csv

```diff
@@ -1,45 +1,45 @@
 metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric
-purefb.info,gauge,,,,FlashBlade system information,0,purefb,info,

```

Comment:
Just make sure you're not removing any metrics but only moving them around. If you remove a metric, that's a breaking change and should be documented as such in the changelog/major version bump.

Reply:
thanks for the feedback, i've updated to a major version 2.0.0 and added the requirements for the latest version of our OpenMetrics Exporter v1.0.11. 

Most of the metrics were just moved around to ensure that they were in alphabetical order, some were added/removed and noted in the changelog.

Reply:
sg thanks!